packages/feedback/src/app/app.component.scss

```diff
@@ -0,0 +1,16 @@
+.app-container {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+.page-container {
+  min-height: 600px;
+}
+
+op-feedback-panel {
```

Comment:
Is this element defined/used anywhere in the code? If not, then this ruleset is not needed.

Same for the above 2 rulesets from this file.

Reply:
Okay